idea-plugin/examples/desktop-project/build.gradle.kts

```
@@ -3,7 +3,8 @@ import org.jetbrains.compose.compose
 plugins {
     // __KOTLIN_COMPOSE_VERSION__
     kotlin("jvm") version "1.5.10"
-    id("org.jetbrains.compose") version "0.0.0-non-interactive-preview-build89-4"
+    // __LATEST_COMPOSE_RELEASE_VERSION__
```

```suggestion
    // __LATEST_COMPOSE_FOR_DESKTOP_RELEASE_VERSION__
```

Would this suggestion work?

Compose could mean anything from Jetpack Compose or Compose for Desktop. Also in this context, it would mean  Compose for Desktop.

Another suggestion would be to rather say `__LATEST_COMPOSE_JB_RELEASE_VERSION__`